purchase_analytic/readme/DESCRIPTION.rst

```diff
@@ -3,3 +3,9 @@ This module add analytic distribution on purchase order.
 
 If all lines of the purchase order have the same analytic distribution, the analytic distribution on the purchase order is automatically set with this value.
 If a analytic distribution is set on the purchase order, all lines of the purchase will take this value.
+
+Limitation
+~~~~~~~~~~
+
+In order to make analytic distribution of the line takes the default value from the corresponding field of purchase.order, it is necessary to save the purchase
+order after setting up the analytic distribution.
```

Comment:
@AungKoKoLin1997 I can't reproduce the behavior of this description in my environment.  Can you?

Reply:
This limitation was before removing analytic group from analytic_distribution field. I will remove this.